### PR TITLE
Improved catalogue search bar styling

### PR DESCRIPTION
**Pull request by kwunkiuma**
image

### CODE REVIEW

**Review by FridaTveit — COMMENTED** [review]
Is this to improve what it looks like? If so, could you please add a screenshot? 🙂

**Comment by kwunkiuma** [comment]
I added the screenshot a few seconds after requesting review, maybe it hadn't processed yet.

File: Bookish.Web/Views/Home/Catalogue.cshtml [hunk 1]
```diff
@@ -14,8 +14,14 @@
 else
 {
     <form asp-action="Catalogue">
-        <input type="text" name="filter"/>
-        <input type="submit" value="Search" />
+        <div class="form-row">
+            <div class="form-group col-md-11">
+                <input type="text" name="filter" class="form-control" />
```
Comment (FridaTveit) [inline Bookish.Web/Views/Home/Catalogue.cshtml]: Might be worth having some placeholder text? 🙂

**Review by FridaTveit — COMMENTED** [review]
One small suggestion otherwise looks good 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**